src/BlockHorizons/BlockPets/pets/BasePet.php

```diff
@@ -59,6 +59,13 @@ abstract class BasePet extends Living {
 	const NETWORK_ID      = -1;
 	const NETWORK_NAME    = null;
 	const NETWORK_ORIG_ID = null;
+	
+	public const ALLAY = "minecraft:allay";
+	public const AXOLOTL = "minecraft:axolotl";
+	public const BEE = "minecraft:bee";
+	public const FOX = "minecraft:fox";
+	public const GOAT = "minecraft:goat";
+	public const WARDEN = "minecraft:warden";
```

Comment:
It would be better if these constants were moved to a separate class (interface preferred e.g. `PetIds`) and then `BasePet` implementing that class.